plugins/modules/aci_l3out_static_routes.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l3out_static_routes
+short_description: Manage External Subnet objects (l3extSubnet:extsubnet)
+description:
+- Manage External Subnet objects (l3extSubnet:extsubnet)
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  logical_node:
+    description:
+    - Name of an existing logical node profile.
+    type: str
+  fabric_node:
+    description:
+    - Name of an existing fabric node.
+    type: str
+  static_route:
+    description:
+    - Configure IP and next hop IP for the routed outside network.
+    type: str
+    aliases: [ address, ip ]
+  state:
+    description:
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    default: query
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+
+notes:
+- The C(tenant) and C(domain) and C(vrf) used must exist before using this module in your playbook.
```

Comment:
Same as other we should specify what is needed here so l3out for sure and probably node profile and node.